app/serializers/user_serializer.rb

```diff
@@ -8,4 +8,22 @@ def salary
   def photo
     self.object.photo
   end
+
+  def pending_vacations
```

Comment:
I believe we have different types of vacations. I think it makes sense to have more counters:
day offs pending, day offs approved, sick leave pending, ... etc.